waspc/src/Wasp/Node/Version.hs

```diff
@@ -76,7 +76,7 @@ readCommandOutput :: String -> [String] -> IO (Either ErrorMessage String)
 readCommandOutput commandName commandArgs = do
   commandResult <-
     catchIOError
-      (Right <$> readProcessWithExitCode commandName commandArgs "")
+      (Right <$> readCreateProcessWithExitCode (shell fullCommand) "")
```

Comment:
Hm same question here as above, do we need to somehow do something with arguments? fullCommand was used only for printing so that was fine, but now we are passing it to shell.